app/Controllers/Http/ProfileController.ts

```diff
@@ -0,0 +1,76 @@
+// import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import Profile from "App/Models/Profile"
+import User from "App/Models/User"
+import UserProfileValidator from "App/Validators/UserProfileValidator"
+import {DateTime} from "luxon"
+
+export default class ProfileController {
+
+    public async view(){
+        const profiles=await Profile.all()
```

Comment:
this API will return all the users profile, make sure it should return only current logged in user profile